autorecovery/autorecovery.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import boto3
+import os
+
+def add_spof_cloudwatch_alarm(instance_list):
+
+    dimensions = []
+    for instance in instance_list:
+
+        cwclient = boto3.client('cloudwatch', region_name=region)
+        response = cwclient.put_metric_alarm(
+            AlarmName='SnowflakeFailure'+instance,
+            AlarmDescription='Alarm that triggers if a SPOF instance is has a failure',
+            ActionsEnabled=True,
+            AlarmActions=[
+                'arn:aws:swf:'+region+':'+account_id+':action/actions/AWS_EC2.InstanceId.Recovery/1.0'
```

Comment:
good point, I'll get this fixed. I think it may be possible to pull it from aws metadata.